### PR TITLE
5 fix templates

### PR DESCRIPTION
**Pull request by mbag**
Please do code review and merge into master.

### CODE REVIEW

File: doc/api/source/_theme/sphinx_rtd_theme/layout.html [hunk 1]
```diff
@@ -121,6 +121,7 @@
 
         <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
           {% block menu %}
+            <a href="{{ pathto('http-routingtable') }}" rel="nofollow">API routing table</a>
```
Comment (dn0) [inline doc/api/source/_theme/sphinx_rtd_theme/layout.html]: It is OK, but it does not show that you are on the current page. I'm suggesting the following change:
```
            <ul>
              <li{% if pagename == 'http-routingtable' %} class="current"{% endif %}>
                <a href="{{ pathto('http-routingtable') }}" rel="nofollow">API routing table</a>
              </li>
            </ul>
```

File: doc/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
 #########
 
+
+2.3.1 (unreleased)
+========================================
+
+Features
+--------
+
+- Added support for nics.*.allowed_ips (multiple IPs per NIC) - `#3 <https://github.com/erigones/esdc-ce/issues/3>`__
```
Comment (dn0) [inline doc/changelog.rst]: This change is not related to this issue. Please remove it

**Comment by mbag** [comment]
Please review changes in commit https://github.com/erigones/esdc-ce/pull/6/commits/f2619cf37881d1ca9d2a79da6f00d5d22f673047
